### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by Tanza3D**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,50 @@
...
+You can use [this script](https://pastebin.com/3XGT5HwE) I've made to quickly switch between folders, keep in mind this'll only work on Arch Linux, so you'll need to edit the script for other distros. - `sudo switch /home/tanza/Documents/orgs/osekai/osekai` 
 
-## Database Setup
+## **Database Setup**
 Create a new user, call it whatever, whatever password, just remember it.
 
-Make a new database called `osekai`, you can name it whatever, but osekai is recommended. It's recommended to make it using utf8mb4_polish_ci
+Make a new database called `osekai`, you can name it whatever, but osekai is recommended. It's recommended to make it using **ufg8mb4_unicode_c1**
 
 Run the `template.sql` file on this table, you can do this by just copying the content into a query tab and running it.
```
Comment (Coppertine) [inline README.md:32]: Need to change to run all files in `db/`

File: README.md [hunk 2]
```diff
@@ -1,32 +1,50 @@
...
 
-## Osekai Tools
+## **Osekai Tools**
 
 Run `git submodule update --init --recursive` to pull all the tool source code, as they're in separate repositories.
 
-## Windows
+## **Windows**
 
 On Windows, SSL will not function with cURL, and Login will not work. To fix, please download [cacert.pem](https://curl.se/docs/caextract.html) and place it in a folder. Then in your `php.ini` under `[curl]` add `curl.cainfo = "C:\php\cacert.pem"` and under `[openssl]` add `openssl.cafile = "C:\php\cacert.pem"`. Should look like so:
```
Comment (Coppertine) [inline README.md:49]: Just recommend user to install and run XAMPP instead. They need to change a spesific file config to point to osekai itself..

**Review by Coppertine — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,32 +1,52 @@
+![](https://raw.githubusercontent.com/Osekai/osekai/main/global/img/cover/medals.jpg)
+# *Osekai*
+Hey there, welcome to the official Osekai repository! We're an open-source website specializing in apps and tools for the rhythm game "osu!"
```
Comment (minusQuantumNeko) [inline README.md:3]:
```suggestion
Hey there, welcome to the official Osekai repository! We're an open-source website specializing in apps and tools for the rhythm game "[osu!](https://osu.ppy.sh/)"
```

File: README.md [hunk 4]
```diff
@@ -1,32 +1,52 @@
...
+If you feel like installing it manually, though, start by setting up apache like you usually would, you can follow a plethora of online tutorials for this. After you're done, just make sure it's using php8.1 and is pointing to the Osekai folder.
+
+You can use [this script](https://pastebin.com/3XGT5HwE) I've made to quickly switch between folders, keep in mind this'll only work on Arch Linux, so you'll need to edit the script for other distros. - `sudo switch /home/tanza/Documents/orgs/osekai/osekai` 
 
-## Database Setup
+## **Database Setup**
 Create a new user, call it whatever, whatever password, just remember it.
 
-Make a new database called `osekai`, you can name it whatever, but osekai is recommended. It's recommended to make it using utf8mb4_polish_ci
+Make a new database called `osekai`, you can name it whatever, but osekai is recommended. It's recommended to make it using **ufg8mb4_unicode_c1**
```
Comment (EXtremeExploit) [inline README.md:32]: i think the name of this collate is wrong, this should be the correct one i think
```suggestion
Make a new database called `osekai`, you can name it whatever, but osekai is recommended. It's recommended to make it using **utf8mb4_unicode_ci**
```

**Review by EXtremeExploit — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Tanza3D) [reply README.md:32]: achh i misspelt utf thank you for pointing that out

**Review by minusQuantumNeko — DISMISSED** [review]

**Review by Coppertine — CHANGES_REQUESTED** [review]
lgtm

**Review by EXtremeExploit — COMMENTED** [review]
yes

**Review by EXtremeExploit — APPROVED** [review]
my dumbass made a comment instead of review

**Pull request merged**